### PR TITLE
Changed exported file name

### LINKED ISSUE #1769: Set assignment name as name for export results file.
Currently, the exported file name is assignment-grades. It would be better to have the name of the assignment instead. 

**Are you working on this? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by arjungarg07**
Fixes #1769 

#### Describe the changes you have made in this PR -
Changed the exported file name from assignment-grades to the name of the assignment instead.
### Screenshots of the changes (If any) -

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by arjungarg07** [comment]
Please review it @tachyons. Thanks.

**Comment by satu0king** [comment]
@arjungarg07 did you try running it?

image

File: app/controllers/grades_controller.rb [hunk 1]
```diff
@@ -40,7 +40,7 @@ def to_csv
     respond_to do |format|
       format.csv do
         send_data Grade.to_csv(params[:assignment_id].to_i),
-                  filename: "assignment-grades.csv"
+                  filename: "#{assignment.name}.csv"
```
Comment (satu0king) [inline app/controllers/grades_controller.rb:43]: assignment is not defined

**Review by satu0king — CHANGES_REQUESTED** [review]

File: app/controllers/grades_controller.rb (hunk 1 above)
Comment (aayush-05) [reply app/controllers/grades_controller.rb:43]: We can rather use assignment id as name. It is already there in the params and wouldn't ever conflict with any existing file

Comment (satu0king) [reply app/controllers/grades_controller.rb:43]: @aayush-05 the user will not understand assignment id. It is just a number. It is important to use assignment name.

**Comment by satu0king** [comment]
@arjungarg07 working on this?

**Comment by arjungarg07** [comment]
I am not able to find the section where this assignment name is displayed as it's somewhere in the simulator projects and only accessed by logging in and I am not able to log in on localhost. Can you guide me on this? @satu0king

**Comment by satu0king** [comment]
@arjungarg07 did u test it?

File: app/controllers/grades_controller.rb (hunk 1 above)
Comment (arjungarg07) [reply app/controllers/grades_controller.rb]: Assignment name is not there in the params and is conflicting with the existing file. What should I do? @satu0king

File: app/controllers/grades_controller.rb [hunk 2]
```diff
@@ -40,7 +40,7 @@ def to_csv
     respond_to do |format|
       format.csv do
         send_data Grade.to_csv(params[:assignment_id].to_i),
-                  filename: "assignment-grades.csv"
+                  filename: "#{@assignment.name}.csv"
```
Comment (satu0king) [inline app/controllers/grades_controller.rb:43]: Just do `Assignment.find(params[:assignment_id]).name` and please test PR. Don't send PR without testing.

**Review by satu0king — CHANGES_REQUESTED** [review]

File: app/controllers/grades_controller.rb (hunk 2 above)
Comment (arjungarg07) [reply app/controllers/grades_controller.rb:43]: Sorry for the inconvenience, I will fix this and also upload screenshots after testing.

**Pull request closed without merging**